src/packages/core/tests/server.test.ts

```diff
@@ -350,6 +392,68 @@ describe("server", () => {
       }
     });
 
+    it("closes even if a persistent http connection is open", async () => {
+      const agent = new http.Agent({
+        keepAlive: true
+      });
+
+      await setup();
+      const jsonRpcJson: any = {
+        jsonrpc: "2.0",
+        id: "1",
+        method: "eth_block_number",
```

Comment:
I think this should be `"eth_blockNumber"`

Reply:
You are absolutely right! Fixed.